### DISCUSSION TITLE
is hardhat-ledger support Trezor hardware wallet?

### DISCUSSION THREAD

**Discussion by brians210** [NomicFoundation/hardhat#4936] [Troubleshooting] [answered]
I am using Trezor and I want to use it to sign or call the contract. I saw hardhat-ledger can support hardware wallet, but is it only support "ledger"? If yes, what should I use for Trezor?

**Comment by kanej** [comment, marked as answer]
Hey @brians210, `hardhat-ledger` does not support trezor.
I am not aware of any plugins that provide trezor support.

**Reply by brians210** [reply]
@kanej so, does ledger and hardhat-ledger support Ronin network?

**Reply by kanej** [reply]
I am not familiar with the Ronin network. If it is EVM compatible at the transaction level and supports Trezor, I would expect that it supports ledger (and so hardhat-ledger). But the Ronin community will have a better understanding of that.
